### PR TITLE
Docker fixes

### LINKED ISSUE #13: Docker not pulling requirements.txt
Ran across a couple issues running in Docker:

- Docker-compose is not executing correctly because the Dockerfile is not referencing the requirements.txt file.
- Some minor changes needed to docker-compose.yml as obs-websocket is not v5 by default

I have fixed to make this work and will submit a PR. @tt2468 can you label the repo with hacktoberfest topic?

### PR DESCRIPTION
**Pull request by airbornelamb**
Closes #13

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ services:
       - API_ADDRESS=0.0.0.0         # Address of this machine
       - API_PORT=4445               # Port you wish to use for API
       - API_KEY=                    # Auth key you wish to set
-      - OBS_URL=ws://127.0.0.1:4444 # WebSocket Connect URL
+      - OBS_URL=ws://127.0.0.1:4455 # WebSocket Connect URL
       - OBS_PASSWORD=               # OBS password, if used
     ports:
       - "4445:4445"             # Set to same value as API_PORT
```
Comment (tt2468) [inline docker-compose.yml:14]: Sticking with the trend of ws + 1, let's use port 4456 by default

Comment (airbornelamb) [reply docker-compose.yml:14]: This should be 4455 because it is set that way by default in OBS versions => 28

Comment (tt2468) [reply docker-compose.yml:14]: I was referring to the usage of port 4445 as API_PORT

**Comment by airbornelamb** [comment]
Added commit to change API port

**Comment by tt2468** [comment]
Superseded by 324ffef0a7b20938c9f4fc90afb019d967755938

**Pull request closed without merging**